redbot/core/core_commands.py

```diff
@@ -976,6 +979,70 @@ async def setdescription(self, ctx: commands.Context, *, description: str = ""):
             ctx.bot.description = description
             await ctx.tick()
 
+    @checks.is_owner()
+    @_set.command(name="storenames")
+    async def storenames(self, ctx: commands.Context):
+        """
+        Toggle whether or not to store user and nick names.
+
+        Default is Off
+        """
+        ctx.bot._store_names = not ctx.bot._store_names
+        await ctx.bot._config.store_names.set(ctx.bot.store_names)
```

Comment:
A DateTime object is being passed here - can't trace where from? I had zero config setup